client/keys/migrate_test.go

```diff
@@ -5,44 +5,38 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/cosmos/cosmos-sdk/client"
+
+	"github.com/stretchr/testify/require"
+
 	"github.com/otiai10/copy"
 	"github.com/stretchr/testify/assert"
-	"github.com/tendermint/tendermint/libs/cli"
 
-	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/crypto/keyring"
 	"github.com/cosmos/cosmos-sdk/testutil"
 )
 
 func Test_runMigrateCmd(t *testing.T) {
-	cmd := AddKeyCommand()
-	_ = testutil.ApplyMockIODiscardOutErr(cmd)
-	cmd.Flags().AddFlagSet(Commands("home").PersistentFlags())
-
 	kbHome := t.TempDir()
-
 	clientCtx := client.Context{}.WithKeyringDir(kbHome)
 	ctx := context.WithValue(context.Background(), client.ClientContextKey, &clientCtx)
 
-	copy.Copy("testdata", kbHome)
-	cmd.SetArgs([]string{
-		"keyname1",
-		fmt.Sprintf("--%s=%s", cli.OutputFlag, OutputFormatText),
-		fmt.Sprintf("--%s=%s", flags.FlagKeyringBackend, keyring.BackendTest),
-	})
-	assert.NoError(t, cmd.ExecuteContext(ctx))
+	require.NoError(t, copy.Copy("testdata", kbHome))
 
-	cmd = MigrateCommand()
+	cmd := MigrateCommand()
 	cmd.Flags().AddFlagSet(Commands("home").PersistentFlags())
-	mockIn := testutil.ApplyMockIODiscardOutErr(cmd)
+	//mockIn := testutil.ApplyMockIODiscardOutErr(cmd)
+	mockIn, mockOut := testutil.ApplyMockIO(cmd)
 
 	cmd.SetArgs([]string{
-		fmt.Sprintf("--%s=%s", flags.FlagHome, kbHome),
+		kbHome,
+		//fmt.Sprintf("--%s=%s", flags.FlagHome, kbHome),
 		fmt.Sprintf("--%s=true", flags.FlagDryRun),
 		fmt.Sprintf("--%s=%s", flags.FlagKeyringBackend, keyring.BackendTest),
 	})
 
 	mockIn.Reset("test1234\ntest1234\n")
+	t.Log(mockOut.String())
```

Comment:
@sahith-narahari I need your help with the line above. I don't know the password you used to secure the old keybase's password (testdata)

Reply:
I used "123456789", looks like the keys are replaced now though

Reply:
Yes, I replaced the testdata :+1: